project_national/fact_sheets_category_3.yml

```diff
@@ -0,0 +1,354 @@
+
+    reporting_measures:
+      - measure_dir_name: QOIReport
+
+    server_directory_cleanup:
+      retain_in_idf: false
+      retain_schedules_csv: false
+
+### -- Upgrades ---
+upgrades:
+  - upgrade_name: All ENERGYSTAR Appliances
+    options:
+      - &energystar_refrigerator
+        option: Refrigerator|EF 19.9
+        apply_logic:
+          - or:
+            - Refrigerator|EF 6.7
+            - Refrigerator|EF 10.2
+            - Refrigerator|EF 10.5
+            - Refrigerator|EF 15.9
+            - Refrigerator|EF 17.6
+        costs:
+          - value: 2003.0 # price1: $1699-1799 for ENERGY STAR Certified 20.9 Cu.Ft. Refrigerator in BestBuy
+                          # price2: $8144-8599 for ENERGY STAR Certified 20.9 Cu.Ft. Refrigerator in Home Depot. This price is too high. Therefore, will not refer these prices. 
+                          # 2003.0=1749*1.07+94*1.4, price*1.07 sales tax + 94*1.4 labor. 1749=(1699+1799)/2
+                          # Another option: $938. $670 for EF 19.9 20.9 Cu.Ft. Refrigerator in REMDB, then apply 1.4 inflation factor (Jan 2010 to Apr 2023)
+            multiplier: Fixed (1)
+        lifetime: 17.4 # 17.4 from REMDB https://remdb.nrel.gov/measures?group_id=4&component_type_id=278&actionId=1044&bcId=4261
+                       # 15 from IEA https://www.eia.gov/analysis/studies/buildings/equipcosts/pdf/appendix-a.pdf
+      - &energystar_extra_refrigerator
+        option: Misc Extra Refrigerator|EF 19.9
+        apply_logic:
+          - or:
+            - Misc Extra Refrigerator|EF 6.9, National Average
+            - Misc Extra Refrigerator|EF 6.9, Average Installed
+            - Misc Extra Refrigerator|EF 15.9, Fed Standard, bottom freezer-reference fridge
+            - Misc Extra Refrigerator|EF 19.8, bottom freezer
+            - Misc Extra Refrigerator|EF 6.7
+            - Misc Extra Refrigerator|EF 10.2
+            - Misc Extra Refrigerator|EF 10.5
+            - Misc Extra Refrigerator|EF 15.9
+            - Misc Extra Refrigerator|EF 17.6
+        costs:
+          - value: 2003.0 # price1: $1699-1799 for ENERGY STAR Certified 20.9 Cu.Ft. Refrigerator in BestBuy
+                          # price2: $8144-8599 for ENERGY STAR Certified 20.9 Cu.Ft. Refrigerator in Home Depot. This price is too high. Therefore, will not refer these prices. 
+                          # 2003.0=1749*1.07+94*1.4, price*1.07 sales tax + 94*1.4 labor. 1749=(1699+1799)/2
+                          # Another option: $938. $670 for EF 19.9 20.9 Cu.Ft. Refrigerator in REMDB, then apply 1.4 inflation factor (Jan 2010 to Apr 2023)
+            multiplier: Fixed (1)
+        lifetime: 17.4 # 17.4 from REMDB https://remdb.nrel.gov/measures?group_id=4&component_type_id=278&actionId=1044&bcId=4261
+                       # 15 from IEA https://www.eia.gov/analysis/studies/buildings/equipcosts/pdf/appendix-a.pdf
+      - &energystar_clothes_washer
+        option: Clothes Washer|EnergyStar
```

Comment:
It looks like the Clothes Washer|EnergyStar option has a clothes_washer_efficiency of 2.07 in develop and 2.06 in our branch. Did we change it? We should not do that I don't think? Because the baseline uses that option, and presumably the specs are set correctly for baseline. So if we want to do something that is different than what is in baseline we should make a new option. 

But we may want to check with @afontani  on this.

Reply:
In our previous version of options_lookup.tsv, Clothes Washer|EnergyStar option has a clothes_washer_efficiency of 2.06, which is based on the assumption for Residential Clothes Washers (≤ 2.5 cu-ft) . But we decided to use the Residential Clothes Washers, Top-loading (> 2.5 cu-ft)  as reference, which has a clothes_washer_efficiency of 2.07. Therefore, I changed Clothes Washer|EnergyStar option to clothes_washer_efficiency of 2.07.

Reply:
Let's leave baseline as-is (ie 2.07), and make a separate option for this analysis that uses the stated specs (top-loading, 2.06).